documents/write_acls.go

```diff
@@ -84,3 +86,121 @@ func newChangedField(p proofs.Property, leaf *proofs.LeafNode, old bool) changed
 	cf.new = v
 	return cf
 }
+
+// transitionRulesFor returns a copy all the transition rules for the account.
+func (cd *CoreDocument) transitionRulesFor(account identity.DID) (rules []coredocumentpb.TransitionRule) {
```

Comment:
This comment is about naming. I think in cases like this `account` is only relevant from the context of hosting identities. I would use did/id similar here instead of account, as we have account struct as well can be confusing.

Reply:
according to what I described in the tech talk, the `identities` package should be renamed to `accounts` in the future and all accounts stuff moved there. Then above param would look like `account identity.DID`

Reply:
Not sure if accounts as generic term floating around makes sense. In the Multi-tenency perspective sure. But DID take multiple forms like collaborator(when exchanging documents), account(multi-tenency), id(node self id) etc...

I wouldn't rename it to accounts.

Reply:
It can take multiple/generic forms from a `name` perspective, but not from a perspective of a `type`. Account is part of the protocol, same as what an ethereum account means for us. Thats atleast the discussion that lead to naming multi-tenancy to multi account. We can discuss, when the time comes for reorg of code anyways.

Reply:
@vimukthi-git @mikiquantum lets take this offline. This may be not the right place to dicuss this :)